.gitignore

```diff
@@ -20,6 +20,11 @@ covid_xprize/examples/predictors/lstm/models/
 covid_xprize/examples/predictors/lstm/predictions/
 covid_xprize/examples/predictors/lstm/tests/fixtures/OxCGRT_latest.csv
 
+*prescription_*.csv
+*tmp_predictions_for_prescriptions/
+covid_xprize/examples/prescriptors/neat/data/
+covid_xprize/examples/prescriptors/neat/neat-checkpoint-*
+covid_xprize/examples/prescriptors/neat/tmp_prescription.csv
```

Comment:
Do we not need `covid_xprize/validation/data/OxCGRT_latest.csv` also? I can't remember how I ended up with that

Reply:
@dsargent I've added it below. It can be created by a few different things, one of which is neat, when it calls the `cost_generator` in `validation`.